pkg/docker/docker.go

```diff
@@ -563,7 +684,10 @@ func (d *stiDocker) GetAssembleInputFiles(image string) (string, error) {
 }
 
 // getScriptsURL finds a scripts url label in the image metadata
-func getScriptsURL(image *docker.Image) string {
+func getScriptsURL(image *api.Image) string {
+	if image == nil {
+		return ""
+	}
```

Comment:
why do we need this check here but not in other places where we take an image pointer?